src/http/routing.rs

```diff
@@ -663,11 +752,27 @@ pub enum RouteInfo<'a> {
     CreateEmoji {
         guild_id: u64,
     },
+    CreateFollowupMessage {
+        application_id: u64,
+        interaction_token: &'a str,
+        wait: bool
```

Comment:
```suggestion
        wait: bool,
```
Missing trailing comma.